src/components/DlChip/DlChip.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div
+        v-if="modelValue"
```

Comment:
Please remove the v-model. this can be tracked internally and not with a v-model as its not needed